### PR TITLE
dovetail 008

### PR DESCRIPTION
**Pull request by zkamvar**
This announces the official dates for the Beta Phase

### CODE REVIEW

File: _posts/2022/10/2022-10-05-dovetail-008.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+## Beta Phase To Resume
+
+As we enter into the Q4, I am pleased to announce that the long-delayed Beta
+Phase will commence from 2022-10-17 until 2023-03-13 with one lesson entering
+the beta phase per week. Details of the beta phase and the official table of
+lessons can be found below and [on the Workbench Beta Phase page](https://carpentries.github.io/workbench/beta-phase.html),
+with explanation of what the different stages of the beta phase will bring.
+
+In brief, the beta phase will run for 12 weeks[^weeks] for each lesson during 
+which a link will appear on the lesson which leads to the same lesson using The
```
Comment (tobyhodges) [inline _posts/2022/10/2022-10-05-dovetail-008.md]:
```suggestion
which a link will appear on the lesson, leading to the same lesson using The
```

File: _posts/2022/10/2022-10-05-dovetail-008.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+Carpentries Workbench. During the beta phase, **we will be asking for feedback
+about your experience with The Workbench in short assessments** and then invite
+participants to an optional focus group to gather qualitative feedback. The
+feedback we receive from the assessments will make sure that The Workbench
+rollout in 2023-Q2 is smooth.
```
Comment (tobyhodges) [inline _posts/2022/10/2022-10-05-dovetail-008.md]:
```suggestion
about your experience with The Workbench in short assessments** and will then invite
participants to an optional focus group to gather qualitative feedback. The
feedback we receive from the assessments and focus group will make sure that The Workbench
rollout in 2023-Q2 is smooth. 
```

File: _posts/2022/10/2022-10-05-dovetail-008.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+## Tips and Tricks for Using The Workbench
+
+In many of our lessons, we often use errors as teachable moments. Because of
+this all R lessons will still render a website even if there are errors in the
+examples, but this is a double-edged sword when unexpected errors happen. If you
+want to prevent unexpected errors from building a lesson, you can 
+[add `fail_on_error: true` to your `config.yaml` file](https://carpentries.github.io/sandpaper/reference/set_config.html#optional-keypairs-known-by-sandpaper)
+and any chunks without an `error = TRUE` option will cause the build to fail if
+there is an error.
```
Comment (tobyhodges) [inline _posts/2022/10/2022-10-05-dovetail-008.md:125]: ⭐

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Looks pretty good. I have a couple specific suggestions, and also ask that you choose between "the Beta" and "the beta" and use one consistently throughout.

**Review by acrall — APPROVED** [review]
All looks great! Thanks, @zkamvar

**Pull request merged**